core/templates/dev/head/pages/learner_dashboard/LearnerDashboard.js

```diff
@@ -49,30 +55,51 @@ oppia.constant('HUMAN_READABLE_SUBSCRIPTION_SORT_BY_KEYS', {
   IMPACT: 'Impact'
 });
 
+oppia.constant('HUMAN_READABLE_FEEDBACK_THREADS_SORT_BY_KEYS', {
```

Comment:
Instead of separating this from the above, perhaps it might be better to have a constant that looks something like this:

```
{
  LAST_UPDATED: {
    key: 'last_updated',
    i18nId: 'I18N_...'
  },
  ...
}
```

so that the two things automatically stay in sync. What do you think? Also I would specifically refer to i18n IDs here rather than "human readable string" because 'I18N_DASHBOARD_...' is not a human-readable string.